Automattic/wp-calypso
# Command Palette: Add prompt icon to the command palette's search bar

Related to
- https://github.com/Automattic/dotcom-forge/issues/7501

## Proposed Changes

* Refactored the icon structure so that the command palette package owns the icon
* The icon is exported so that other clients can consume it.
* If the icon changes, all clients using it will update accordingly.

## Why are these changes being made?


* Requested by stakeholder walkthrough.

## Testing Instructions



* Open the Calypso live link
* Navigate to `/sites`
* The icon displayed at the bottom should be the prompt button
* Open the command palette (cmd+k)
* You should see the prompt icon instead of the magnifying glass icon.
* Apply these changes to your sandbox and test on wp-admin (just the part where you open the command palette).
 
## Pre-merge Checklist
